src/styles/_variables.scss

```diff
@@ -0,0 +1,49 @@
+// Colour variables created in AEG-25
+
+// Colour - Text
+$text-primary: #0f2544;
+$text-offset: #eeebe8;
+$text-emphasis: #fe5c32;
+
+// Resp Design
+$screen-mobile: 280px;
+$screen-tablet: 768px;
+// tablet vertical and horizontal?
+$screen-desktop: 1280px;
+
+// Margins
+$margin-mobile-xs: .5rem;
+$margin-mobile-s: .75rem;
+$margin-mobile-m: 1rem;
+$margin-mobile-l: 1.5rem;
+
+$margin-tablet-xs: .5rem;

```

Comment:
Curious on your take of the benefit of using rem for margins and padding. I was taught that you should use rem and em for font-only and if necessary for paddings/margins around text-itself so that this scales.  My understanding for _using_ rem is so that if a user manually changes their font-size, that it adjusts the margins/padding as well so it can be better for accessibility. 

I had typically stuck to using px or if necessary vw/%. Usually with the premise that regardless if the user was at 320 or 430 I don't want the margins to change and if the font-size is increased/decreased, margins changing might affect the spacing and positioning of elements (although much less likely with flex etc).

I'm perfectly open to both approaches but interested on the use cases.

Reply:
Also happy to look at revisiting the margin/padding scaling so that if we do decide to change the root font for our media queries that they scale accordingly and reduce the amount of overlap.